assets/models/system/microsoft-phi-2/model.yaml

```diff
@@ -1,6 +1,6 @@
 path:
   container_name: models
-  container_path: huggingface/microsoft-phi-2/1702485256/mlflow_model_folder
+  container_path: huggingface/microsoft-phi-2/1704358138/mlflow_model_folder
```

Comment:
what was the previous value of `eos_token_id` that has been updated to a new value now?

Reply:
Will that affect any other scenario?

Reply:
it was missing from config.json, That will not affect other scenarios.

Reply:
Sounds good